### PR TITLE
DTSPO-12024 - Update helm repo

### PR DESCRIPTION
**Pull request by endakelly**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSPO-12024

### Change description ###
Add codeowners and renovate config
Update helm repo to enable renovate to update chart dependencies

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by timja — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @hmcts/idam
```
Comment (timja) [inline .github/CODEOWNERS:1]: unsure if this is the right team, is this chart actually in use? looks to be done by the old family public law team

Comment (endakelly) [reply .github/CODEOWNERS:1]: Well it hasn't been updated in 3 years so I'd imagine it's not in use but I'll do a search to see if anything is consuming it. Any advice on a person or slack channel that can be contacted for info?

Comment (timja) [reply .github/CODEOWNERS:1]: see if its in use and if it isn't then archive it, check the contact channel for any team that is using it if you find it

**Comment by endakelly** [comment]
Search on github suggests it's not being consumed. Will close this and archive the repo

**Pull request closed without merging**